File: Demo-LLP-MIP/Demo-LLP-MIP-Python/v1/isign/argument/query/QueryParams.py
class QueryParams:
    def __init__(self):
        self.mch_id = None
        self.sub_mchid = None
        self.user_id = None

    def to_dict(self):
        return {
            "mch_id": self.mch_id,
            "sub_mchid": self.sub_mchid,
            "user_id": self.user_id
        }

File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/v1/isign/argument/query/QueryParams.php
<?php


class QueryParams
{
    public $mch_id;
    public $sub_mchid;
    public $user_id;

    public function __construct()
    {
        $this->mch_id = null;
        $this->sub_mchid = null;
        $this->user_id = null;
    }

    public function to_dict() {
        $data = [];
        foreach ($this as $key => $value) {
            if ($value !== null) {
                $data[$key] = $value;
            }
        }
        return json_encode($data);
    }
}